test/functional/multiwallet.py

```diff
@@ -18,6 +20,21 @@ def __init__(self):
         self.extra_args = [['-wallet=w1', '-wallet=w2', '-wallet=w3']]
 
     def run_test(self):
+        self.stop_node(0)
```

Comment:
I'm used to see the success tests at bottom. Can change if it's better as you suggest.